typescript/infra/config/environments/mainnet3/warp/configGetters/getArbitrumBaseEthereumRadixUSDCWarpConfig.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  ChainMap,
+  HypTokenRouterConfig,
+  IsmType,
+  OwnableConfig,
+  RoutingIsmConfig,
+  TokenType,
+  buildAggregationIsmConfigs,
+  defaultMultisigConfigs,
+} from '@hyperlane-xyz/sdk';
+
+import {
+  RouterConfigWithoutOwner,
+  tokens,
+} from '../../../../../src/config/warp.js';
+
+import { getUSDCRebalancingBridgesConfigFor } from './utils.js';
+
+const getIsm = (local: keyof typeof owners): RoutingIsmConfig => {
+  return {
+    type: IsmType.FALLBACK_ROUTING,
+    owner: owners[local],
+    domains: buildAggregationIsmConfigs(
+      local,
+      ['radix'],
+      defaultMultisigConfigs,
+    ),
+  };
+};
+
+const owners = {
+  ethereum: '0xA365Bf3Da1f1B01E2a80f9261Ec717B305b2Eb8F',
+  arbitrum: '0xA365Bf3Da1f1B01E2a80f9261Ec717B305b2Eb8F',
+  base: '0xA365Bf3Da1f1B01E2a80f9261Ec717B305b2Eb8F',
+  bnb: '0xA365Bf3Da1f1B01E2a80f9261Ec717B305b2Eb8F',
+  radix: 'account_rdx1280taxhhnuek02y59yapsg4kjtux954qkyufpwmy4dlfcxdrjzr7fj',
+};
+
+export const getArbitrumBaseEthereumRadixUSDCWarpConfig = async (
+  routerConfig: ChainMap<RouterConfigWithoutOwner>,
+  abacusWorksEnvOwnerConfig: ChainMap<OwnableConfig>,
+): Promise<ChainMap<HypTokenRouterConfig>> => {
+  const rebalancingConfig = getUSDCRebalancingBridgesConfigFor(
+    Object.keys(owners),
+  );
+
+  const ethereum: HypTokenRouterConfig = {
+    ...routerConfig.ethereum,
+    decimals: 6,
+    interchainSecurityModule: getIsm('ethereum'),
+    owner: owners.ethereum,
+    proxyAdmin: {
```

Comment:
Remove this override, it will by default use the top level owner, and a smaller visual diff is easier to reason about